.env.example

```
@@ -0,0 +1,3 @@
+# Context7 MCP Configuration
+# Get your API key from https://context7.com (optional but recommended for higher rate limits)
+CONTEXT7_API_KEY=ctx7sk-e77dbb74-cd45-4ae6-8564-f9545552f58c
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)